### PR TITLE
Facebook: adding mark_seen and typing_on sender actions

### PR DESCRIPTION
**Pull request by shasha79**
**Proposed changes**:
- add mark_seen and typing_on actions (see details here: https://github.com/rehabstudio/fbmessenger#sender-actions)

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 The assigned reviewer will take a look at it as soon as possible✨

File: rasa/core/channels/facebook.py [hunk 1]
```diff
@@ -107,11 +108,12 @@ def _is_quick_reply_message(message: Dict[Text, Any]) -> bool:
         self, text: Text, sender_id: Text, metadata: Optional[Dict[Text, Any]]
     ) -> None:
         """Pass on the text to the dialogue engine for processing."""
-
+        self.client.send_action(SenderAction(sender_action='mark_seen').to_dict(), sender_id)
```
Comment (erohmensing) [inline rasa/core/channels/facebook.py]:
```suggestion

        self.client.send_action(SenderAction(sender_action='mark_seen').to_dict(), sender_id)
```

File: rasa/core/channels/facebook.py [hunk 2]
```diff
@@ -107,11 +108,12 @@ def _is_quick_reply_message(message: Dict[Text, Any]) -> bool:
...
         out_channel = MessengerBot(self.client)
```
Comment (erohmensing) [inline rasa/core/channels/facebook.py]: Is there a way we can use the `MessengerBot` to send these actions instead of `self.client` directly?

**Comment by erohmensing** [comment]
@shasha79 are you still interested in implementing this?

**Comment by tmbo** [comment]
Will close this one as it is stale, the feature has been added on master

**Pull request closed without merging**